sections/announcement-bar.liquid

```diff
@@ -115,6 +128,87 @@
         </div>
       </slideshow-component>
     {%- endif -%}
+    <div class="localization-wrapper">
+      {%- if section.settings.enable_country_selector and localization.available_countries.size > 1 -%}
+        <noscript class="small-hide medium-hide">
+          {%- form 'localization', id: 'HeaderCountryMobileFormNoScript', class: 'localization-form' -%}
```

Comment:
Here the ID needs to be different just in case, since this is already the ID used in `header.liquid`. There is a chance to have duplicate IDs. So let's do something like: 
```suggestion
          {%- form 'localization', id: 'AnnouncementCountryMobileFormNoScript', class: 'localization-form' -%}
```